<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Perfil</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: rgb(122, 82, 208);

}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>560</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>560</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="nomePerfil">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>381</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Armando Luz Borges</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="perfilConta">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>190</y>
         <width>291</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Conta: </string>
       </property>
      </widget>
      <widget class="QLabel" name="emailPerfil">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>230</y>
         <width>301</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>E-mail: </string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_back_perfil">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 18pt &quot;Verdana&quot; ;
	background-color: rgb(255, 255, 255);
	border: rgb(255, 255, 255);
	color: rgb(122, 82, 208);
}

QPushButton:pressed{
	color: rgb(0, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonExit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>310</y>
         <width>280</width>
         <height>50</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 8pt &quot;Verdana&quot; ;
	border: 2px solid rgb(122, 82, 208);
	color: rgb(122, 82, 208);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(255, 255, 255);
}

QPushButton:pressed{
	background-color: rgb(122, 82, 208);
	color: rgb(0, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>SAIR</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
